config/horizon.json

```
@@ -43,6 +43,7 @@
 		"google-my-business": false,
 		"help": true,
 		"home/layout-dev": true,
+		"hosting-server-settings-enhancements": false,
```

```suggestion
		"hosting-server-settings-enhancements": true,
```
See my other comment